spec/support/factory_girl.rb

```diff
@@ -0,0 +1,3 @@
+RSpec.configure do |config|
+      config.include FactoryGirl::Syntax::Methods
```

Comment:
OK, ba je trouve ça bizzare de mettre de la conf RSpec dans support alors qu'il y a des fichiers de conf pour ça... dans `config` ...;)
J'ai probablement loupé un truc...
(any comment is welcome !)